ai_bridge/security/ir.model.access.csv

```diff
@@ -0,0 +1,4 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_ai_bridge_user,ai_bridge.user,model_ai_bridge,base.group_user,1,0,0,0
```

Comment:
A registration rule would be missing depending on the configured company

Reply:
I understand you ask for company permissions. that makes sense and I will add it.